docs/topics/Score-Modifications.md

```diff
@@ -21,8 +21,4 @@ This data includes:
 
 The o!TR team manipulates raw match data in the following ways:
 
-* Multiplies all score values that have the [`EZ` modifier](https://osu.ppy.sh/wiki/en/Gameplay/Game_modifier/Easy) by **1.75x**.
-
-## Removal of user information
```

Comment:
is there a reason this was removed?

Reply:
The privacy statement is the only thing that's gone and I wanted to put that in another page later.